test/dummy/app/models/article.rb

```diff
@@ -1,2 +1,11 @@
 class Article < ActiveRecord::Base
+  belongs_to :admin_user
+
+  def computed
+    "Not orderable"
+  end
```

Comment:
Not sure how to do that. Monkey patch the class and service? This is used in integration tests.